magefiles/utils.go

```diff
@@ -150,3 +165,120 @@ func renderTemplate(destination, templatePath string, templateData interface{},
 
 	return nil
 }
+
+// helper function used for testing jvm-build-service PRs
+func overridePipelineBundle() error {
+	const buildTemplatesBundleKustomizeUrl = "https://raw.githubusercontent.com/redhat-appstudio/infra-deployments/main/components/build-templates/kustomization.yaml"
+	const bundleName = "hacbs_build_bundle"
+	var newReqprocessorImage = os.Getenv("JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE")
+
+	const imagePushRepo = "quay.io/redhat-appstudio-qe/test-images"
```

Comment:
nit: Should we comment what this repo is more explicitly used or intent for? I'm not familiar with this quay repo myself and others maybe be curious as well